packages/react/src/components/Chat/ChatMessage.tsx

```diff
@@ -178,6 +184,33 @@ class ChatMessage extends UIComponent<WithAsProp<ChatMessageProps>, ChatMessageS
     _.invoke(this.props, 'onBlur', e, this.props)
   }
 
+  renderActionMenu(
+    actionMenu: ChatMessageProps['actionMenu'],
+    styles: ComponentSlotStylesPrepared,
+  ) {
+    const actionMenuElement = Menu.create(actionMenu, {
+      defaultProps: {
+        [IS_FOCUSABLE_ATTRIBUTE]: true,
+        accessibility: menuAsToolbarBehavior,
+        className: ChatMessage.slotClassNames.actionMenu,
+        styles: styles.actionMenu,
+      },
+    })
+
+    if (!actionMenuElement) {
+      return actionMenuElement
+    }
+
+    return (
+      <Popper targetRef={this.state.messageDomNode} position="above" align="end">
+        {({ scheduleUpdate }) => {
+          this.updateActionsMenuPosition = scheduleUpdate
```

Comment:
it is necessary to trigger position updates for the cases when actions menu appear, as it has width of `0` when is hidden